wgpu-types/src/lib.rs

```diff
@@ -5107,6 +5107,16 @@ pub struct SurfaceConfiguration<V> {
     /// AutoNoVsync will gracefully do a designed sets of fallbacks if their primary modes are
     /// unsupported.
     pub present_mode: PresentMode,
+    /// Desired maximum number of frames that the presentation engine should queue in advance.
+    ///
+    /// Defaults to 2 when created via `wgpu::Surface::get_default_config`.
+    /// Recommended to use 2 (or higher) for high throughput, 1 for low latency.
```

Comment:
Could we clarify exactly what "high throughput" means?
Maybe it just isn't clear to me because I'm clueless on the topic ...

Reply:
Hmm I'll try to expand a bit on it, but it's intentionally handwavy :/

Reply:
rewrote that section now to be a lot more precise :)